package.json

```diff
@@ -99,7 +99,7 @@
     "jsonfile": "^6.1.0",
     "lilconfig": "^3.1.3",
     "lodash": "^4.17.21",
-    "ora": "^8.1.1",
+    "ora": "5.4.1",
```

Comment:
note: locking ora to a pre ESM release to avoid import issues and incompatibility with pre v30 of jest